coalib/misc/Shell.py

```diff
@@ -1,10 +1,7 @@
-from contextlib import contextmanager
-import functools
 import shlex
-from subprocess import PIPE, Popen, call, DEVNULL
-
-
-call_without_output = functools.partial(call, stdout=DEVNULL, stderr=DEVNULL)
+from contextlib import contextmanager
+from subprocess import PIPE
+from subprocess import Popen
```

Comment:
@nemaniarjun Yeah precisely that. So is the code fine then? Because it did clear all the tests.